src/components/account/details/DetailsPageMain.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+
+import EditAccountDetails from './EditAccountDetails'
+import ChangePassword from './ChangePassword'
+import Subscribe from './Subscribe'
+import EditCompany from './EditCompany'
+import CancelAccount from './CancelAccount'
+
+import '../../../styles/account/details/details_page_main.css'
+
+class DetailsPageMain extends Component {
+  render() {
+    return (
+      <div>
+        <div className="page-main">
+          <div className="header">
+            <h1 className="page-main__title">Account details</h1>
```

Comment:
This is the BEM CSS naming method, it hasn't been enforced project wide yet, but @jrob is a fan...so I have been using this on any new files I create...here's more info below:
http://getbem.com/naming/